.changeset/giant-dogs-dress.md

```diff
@@ -0,0 +1,13 @@
+---
+'@project44-manifest/react-action-card': minor
+'@project44-manifest/react-card': minor
+'@project44-manifest/react-dropdown': minor
```

Comment:
it always perplexes me why these packages get roped into `yarn changeset`, but I'm afraid to modify this as it is autogenerated

Reply:
its safe to manually remove, definitely should not be adding dropdown